tests/test_approval_modules_snapshots/cdf_apm_simple_data_model.yaml

```diff
@@ -356,7 +362,6 @@ Transformation:
   ignoreNullFields: true
   isPublic: true
   name: asset:oid:workmate:apm_simple:load_asset2children
-  ownerIsCurrentUser: true
```

Comment:
This is a server set property, which was set by a default in the read class  which is no longer there.